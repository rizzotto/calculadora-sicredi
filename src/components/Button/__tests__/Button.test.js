import React from "react";
import renderer from "react-test-renderer";

import { Typography } from "@material-ui/core";

import Button from "../index.js";

it("renders button correctly", () => {
    const tree = renderer
        .create(
            <Button>
                <Typography>Test</Typography>
            </Button>
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
