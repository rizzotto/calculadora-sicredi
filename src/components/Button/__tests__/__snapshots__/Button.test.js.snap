// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders button correctly 1`] = `
<button
  className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-lightGreen-2 makeStyles-content-3 MuiButton-containedPrimary"
  disabled={false}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  <span
    className="MuiButton-label"
  >
    <p
      className="MuiTypography-root MuiTypography-body1"
    >
      Test
    </p>
  </span>
</button>
`;
